eng/Versions.props

```diff
@@ -7,7 +7,7 @@
     <MinorVersion>8</MinorVersion>
     <!-- Always use shipping version instead of dummy version -->
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
-    <PreReleaseVersionLabel>preview5</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
```

Comment:
Do we want to call this `rc1` just to be extra cautious?

Reply:
Artak just told me this.  Will fix it.